index.html

```diff
@@ -80,13 +87,13 @@ <h2 class="title">Tonic</h2>
                             <button type="button" class="pro">javascript</button>                               
                         </div>
                         <button type="button" class="see_button">See Project</button>
-                    </div>                
+                    </div>                                  
             </article>
 <!--card_2 starts from here-->
-            <article class="grid_container">
-                <img id="picture2" class="grid_elements" src="./assets/pictures/picture2.jpg" alt=""/>
-                    <div class="grid_elements">                       
-                        <h2 class="title">Multi-Post<br>Stories</h2>
+            <article class="grid_container" id="grid_container_D_2">
```

Comment:
- [ ] You are almost there! Let's fix some little details to achieve the design from the Figma template:

|Template|Your design|
|---|---|
|![Captura de pantalla 2022-06-02 a la(s) 10 38 38 p  m](https://user-images.githubusercontent.com/83717285/171782728-31a732b3-a65c-43b5-bdfc-712f9bc2c388.png)|![Captura de pantalla 2022-06-02 a la(s) 10 37 22 p  m](https://user-images.githubusercontent.com/83717285/171782707-82728bfb-1f54-4304-ad7d-9cde98a5f3c9.png)|

- [ ] It would be great to add some margin (left and right, top and bottom) to the cards like in the Figma template.

- [ ] Let's add more vertical space between "HTML CSS Javascript" and the button.

- [ ] I would suggest adding the button hover state provided in the original template:

![Captura de pantalla 2022-06-02 a la(s) 11 01 19 p  m](https://user-images.githubusercontent.com/83717285/171783561-05c91934-7383-4b02-8612-700ff2f16a9a.png)